tests/test_async_gen.py

```diff
@@ -90,13 +86,12 @@ async def my_stream(cr: ConsumerRecord, stream: Stream):
 
         async with my_stream as stream_flow:
             # It is an async generator so it should not be an asyncio.Task
-            assert not my_stream._consumer_task
+            # assert not my_stream._consumer_task
```

Comment:
should this comment be left in

Reply:
It should be removed. All the `_consumer_task` have been removed